client/src/partials/patient_edit/edit_patient.html

```diff
@@ -0,0 +1,227 @@
+                  <label for="email">{{ "PATIENT_REG.AVENUE" | translate }}</label>
+                  <input type="text" class="form-bhima" id="avenue" ng-model="patient.addr_1">
+                </div>
+
+                <div class="form-group">
+                  <label for="parent">{{ "PATIENT_REG.PARENT_NAME" | translate }}</label>
+                  <input type="text" class="form-bhima" id="father_name" ng-model="patient.father_name">
+                </div>
+
+                <div class="form-group">
+                  <label>{{ "PATIENT_REG.MOTHER_NAME" | translate }}</label>
+                  <input type="text" class="form-bhima" id="mother_name" ng-model="patient.mother_name">
+                </div>
+
+                <div class="form-group">
+                  <label for="religion">{{ "PATIENT_REG.RELIGION" | translate }}</label>
+                  <input type="text" class="form-bhima" id="religion" ng-model="patient.religion">
+                </div>
+
+                <div class="form-group">
+                  <label for="marital">{{ "PATIENT_REG.MARITAL_STATUS" | translate }}</label>
+                  <input type="text" class="form-bhima" id="marital" ng-model="patient.marital_status">
+                </div>
+
+                <div class="form-group">
+                  <label>{{ "PATIENT_REG.PROFESSION" | translate }}</label>
+                  <input type="text" class="form-bhima" id="profession" ng-model="patient.profession">
+                </div>
+
+                <div class="form-group">
+                  <label>{{ "PATIENT_REG.EMPLOYER" | translate }}</label>
+                  <input type="text" class="form-bhima" id="employer" ng-model="patient.employer">
+                </div>
+
+                <div class="form-group">
+                  <label>{{ "PATIENT_REG.SPOUSE" | translate }}</label>
+                  <input type="text" class="form-bhima" id="spouse" ng-model="patient.spouse">
+                </div>
+
+                <div class="form-group">
+                  <label>{{ "PATIENT_REG.SPOUSE_PROFESSION" | translate }}</label>
+                  <input type="text" class="form-bhima" id="spouse_profession" ng-model="patient.spouse_profession">
+                </div>
+
+                <div class="form-group">
+                  <label>{{ "PATIENT_REG.SPOUSE_EMPLOYER" | translate }}</label>
+                  <input type="text" class="form-bhima" id="spouse_employer" ng-model="patient.spouse_employer">
+                </div>
+
+              </form>
+            </div>
+          </div> <!-- End panel -->
+        </div> <!-- End other info row -->
+      </div> <!-- End main column 7 -->
+      <div class="col-xs-5">
+      </div> <!-- End column 5 row -->
+    </div> <!-- end row -->
+  </div>
+</main>
+
+<footer>
+  <div class="pull-right">
+    <a id="submitPatient" ng-click="updatePatient()"
+       ng-disabled="session.failedSessionValidation || location.$invalid || details.$invalid" 
```

Comment:
I do not know how to access the 'location' form from within the 'patientEdit' controller.  I'll fix the other things and then get with you to do this later as a separate pull request.  I have a few other cleanup things that I can do at the same time.